frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -484,21 +483,24 @@ function DashboardInner(props: DashboardProps) {
     >
       {() => (
         <DashboardStyled>
-          {isHeaderVisible && (
-            <DashboardHeaderContainer
-              isFullscreen={isFullscreen}
-              isNightMode={shouldRenderAsNightMode}
-            >
-              <DashboardHeader
-                {...props}
-                onEditingChange={handleSetEditing}
-                setDashboardAttribute={handleSetDashboardAttribute}
-                addParameter={addParameter}
-                parametersWidget={parametersWidget}
-                onSharingClick={handleToggleSharing}
-              />
-            </DashboardHeaderContainer>
-          )}
+          <DashboardHeaderContainer
```

Comment:
Pro tip: Use the hide whitespace option for EZ review.
![image](https://github.com/metabase/metabase/assets/1937582/f8552fd7-abd8-4d62-8a66-bacf73542845)